Index: map.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>require([\"esri/Map\", \"esri/views/MapView\", \"esri/widgets/BasemapToggle\"], function(\r\n    Map,\r\n    MapView,\r\n    BasemapToggle\r\n) {\r\n    // Create the Map with an initial basemap\r\n    var map = new Map({\r\n        basemap: \"hybrid\"\r\n    });\r\n\r\n    // Create the MapView and reference the Map in the instance\r\n    var view = new MapView({\r\n        container: \"viewDiv\",\r\n        map: map,\r\n        center: [12.492373, 41.890251],\r\n        zoom: 6\r\n    });\r\n\r\n    // 1 - Create the widget\r\n    var toggle = new BasemapToggle({\r\n        // 2 - Set properties\r\n        view: view, // view that provides access to the map's 'hybrid' basemap\r\n        nextBasemap: \"osm\" // allows for toggling to the 'osm' basemap\r\n    });\r\n\r\n    // Add widget to the top right corner of the view\r\n    view.ui.add(toggle, \"top-right\");\r\n});
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- map.js	(revision 60fd95964ea78cf1018366bfb5704fe4d529ed9e)
+++ map.js	(date 1602678657524)
@@ -1,4 +1,4 @@
-require(["esri/Map", "esri/views/MapView", "esri/widgets/BasemapToggle"], function(
+require(["esri/Map", "esri/views/MapView", "esri/widgets/BasemapToggle" ], function(
     Map,
     MapView,
     BasemapToggle
